build/build.ignore

```diff
@@ -0,0 +1,8 @@
+.[a-z]*
+docker
+Dockerfile
+docs
+Gruntfile.js
+Jenkinsfile*
+tasks
+test
```

Comment:
This file actually not behave same like `.npmignore`. Please see the list there. But better not to introduce new file and use only `.npmignore`.